Assets/HoloToolkit/Utilities/Shaders/OnTop.shader

```diff
@@ -0,0 +1,55 @@
+﻿Shader "Unlit/OnTop"
```

Comment:
I'm not entirely sold with the name OnTop.  I think this should be a bit more descriptive.